src/Tribe/Tickets.php

```diff
@@ -1491,6 +1506,18 @@ public static function get_event_attendees_count( $post_id, $args = [] ) {
 				return 0;
 			}
 
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: f84976e22963452f8547eb15fe769322</sub>